src/atom/profileAtom.ts

```diff
@@ -4,7 +4,9 @@ const profileAtom = atom({
   key: 'profileAtom',
   default: {
     signedProfileId: null,
-    anonProfileId: null
+    anonProfileId: null,
+    token: null,
+    anonymousToken: null
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding `token` and `anonymousToken` to the `profileAtom` seems like a good idea for managing authentication-related data. However, please ensure that these tokens are not being exposed or stored in an insecure manner. It's important to handle sensitive information such as tokens with care to prevent potential security vulnerabilities.
```diff
    signedProfileId: null,
    anonProfileId: null,
+   token: null,
+   anonymousToken: null
```
Remember to validate these tokens before use and have a mechanism in place to refresh them when they expire.


<!-- This is an auto-generated comment by OSS CodeRabbit -->